api/v1alpha1/orchestrator_types.go

```diff
@@ -20,149 +20,182 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var (
+const (
 	RunningPhase   OrchestratorPhase = "Running"
 	CompletedPhase OrchestratorPhase = "Completed"
 	FailedPhase    OrchestratorPhase = "Failed"
 )
 
 // OrchestratorSpec defines the desired state of Orchestrator
 type OrchestratorSpec struct {
-	SonataFlowOperator   SonataFlowOperator   `json:"sonataFlowOperator,omitempty"`
-	ServerlessOperator   ServerlessOperator   `json:"serverlessOperator,omitempty"`
-	RhdhOperator         RHDHOperator         `json:"rhdhOperator,omitempty"`
-	RhdhPlugins          RHDHPlugins          `json:"rhdhPlugins,omitempty"`
-	PostgresDB           Postgres             `json:"postgres,omitempty"`
-	OrchestratorPlatform OrchestratorPlatform `json:"orchestrator,omitempty"`
-	Tekton               Tekton               `json:"tekton,omitempty"`
-	ArgoCd               ArgoCD               `json:"argocd,omitempty"`
-}
+	// Configuration for ServerlessLogic. Optional
+	ServerlessLogicOperator ServerlessLogicOperator `json:"serverlessLogicOperator,omitempty"`
 
-type Subscription struct {
-	Namespace           string `json:"namespace,omitempty"`
-	Channel             string `json:"channel,omitempty"`
-	InstallPlanApproval string `json:"installPlanApproval,omitempty"`
-	Name                string `json:"name,omitempty"`
-	SourceName          string `json:"sourceName,omitempty"`
-	StartingCSV         string `json:"startingCSV,omitempty"`
-	TargetNamespace     string `json:"targetNamespace,omitempty"`
-}
+	// Configuration for Serverless (K-Native) Operator. Optional
+	ServerlessOperator ServerlessOperator `json:"serverlessOperator,omitempty"`
```

Comment:
in https://github.com/parodos-dev/orchestrator-go-operator/pull/3/files#diff-8098df4b34be91b823f398cf836afed2e2d3e1175761a135f8c3625bb27b3c64R98 you renamed `Serverless` to `Knative` (and in other places too), should it be renamed here too?
The official names is `Serverless` so I think we should stick with it everywhere but I'd like others opinions @jordigilh @masayag @gciavarrini